### ISSUE TITLE
Placeholder: Add a property to hide the button on view mode

### ISSUE THREAD

**Issue by estruyf** [SharePoint/sp-dev-fx-controls-react#206]
#### Category
[x] Enhancement
[ ] Bug
[ ] Question

#### Version

Please specify what version of the library you are using: [latest]

#### Expected / Desired Behavior / Question

Add a property to specify if the placeholder button is hidden on page view mode.

**Comment by estruyf** [comment]
Newly introduced property `hideButton` is available in the latest beta.

Sample usage:

```
<Placeholder iconName='Edit'
                     iconText='Configure your web part'
                     description='Please configure the web part.'
                     buttonLabel='Configure'
                     hideButton={this.props.displayMode === DisplayMode.Read}
                     onConfigure={this._onConfigure} />
```

![read-mode-hide-button](https://user-images.githubusercontent.com/2900833/50296184-2c769180-047a-11e9-999e-7d0e53573d16.gif)

**Comment by estruyf** [comment]
This has been released with the latest `1.11.0` version.
